src/avlizer_confluent.erl

```diff
@@ -303,10 +303,17 @@ terminate(_Reason, _State) ->
 
 %%%_* Internals ================================================================
 
-get_registry_url() ->
-  {ok, #{ schema_registry_url := URL
-        }} = application:get_env(?APPLICATION, ?MODULE),
-  URL.
+get_registry_base_request() ->
+  case application:get_env(?APPLICATION, ?MODULE) of
+    {ok, #{schema_registry_url := URL, schema_registry_sasl := SASL}} -> 
```

Comment:
sys.conf is not the best place to store the secrets. Please consider supporting retrieving this data from a SASL file.